### PR TITLE
Add usage example to README.

### PR DESCRIPTION
**Pull request by hackergrrl**
I had difficulty grokking exactly what this tool did until I read
through the README twice and spent some time playing with the
tool. This PR adds an example usage of sharnessify to hopefully
help the reader understand its operation more easily.

This is also a good opportunity for me to confirm with y'all that I am
indeed using it correctly myself. :)  Thanks for writing this!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,34 @@ That's why the goal of this project is to make it easy to cleanly
...
+```sh
+$ cd /tmp
+
+$ git clone git://github.com/chriscool/sharnessify
+
+$ cd sharnessify
+
+$ sh sharnessify.sh ~/dev/my-project
+
+$ cd ~/dev/my-project
```
Comment (chriscool) [inline README.md]: Yeah. Alternatively after cloning sharnessify you could also do:

```
$ cd ~/dev/my-project
$ /tmp/sharnessify/sharnessify.sh .
```

Comment (chriscool) [reply README.md]: It's also possible to simplify a bit the firsts steps like this:

```
$ cd ~/dev/my-project
$ git clone git://github.com/chriscool/sharnessify /tmp/sharnessify
$ /tmp/sharnessify/sharnessify.sh .
```

Comment (chriscool) [reply README.md]: But maybe your explanations are simpler for someone who don't know git well.
So I am ok to merge them as is.

Comment (hackergrrl) [reply README.md]: I actually like yours a lot better. I'll update.

Comment (hackergrrl) [reply README.md]: Done!

Comment (chriscool) [reply README.md]: Great, thanks! I will merge it.

**Pull request merged**